ci-operator/step-registry/openshift/e2e/libvirt/conf/openshift-e2e-libvirt-conf-commands.sh

```
@@ -311,6 +311,15 @@ EOF
        if [ "${BRANCH}" == "4.21" ]; then
           cat >> "${SHARED_DIR}/excluded_tests" << EOF
 "[sig-node] Pods Extended (pod generation) [Feature:PodObservedGenerationTracking] [FeatureGate:PodObservedGenerationTracking] [Beta] Pod Generation pod observedGeneration field set in pod conditions"
+EOF
+       fi
+       # Skip the HAProxy router idled service test for versions below 4.22 until the fix is backported
+       # The test fails due to missing port 80 configuration, which was fixed in 4.22 by https://github.com/openshift/installer/pull/10548
+       # Backporting to all lower versions requires significant resources and is not feasible at this time
+       # Bug reference: https://redhat.atlassian.net/browse/OCPBUGS-85232
+       if echo ${BRANCH} | awk -F. '{ if ((($1 == 4) && ($2 < 22))) { exit 0 } else { exit 1 } }'; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)